app/controllers/doc_controller.rb

```diff
@@ -6,7 +6,7 @@ class DocController < ApplicationController
   before_filter :set_book, only: [:index]
 
   def index
-    @videos = VIDEOS
+    @videos = Gitscm::VIDEOS
```

Comment:
I wonder why it doesn't break in production but locally and in review app does? Probably some caching

Reply:
yes, confirmed https://git-scm.herokuapp.com/doc